lib/components/T3Renderer/T3Renderer.js

```diff
@@ -44,8 +50,8 @@ export default {
     // render loop of content elements
     return ctx.props.content.map((element, index) =>
       ctx.props.frame && element.appearance.frameClass !== 'none'
-        ? renderFrame(element, index)
-        : renderComponent(element, index)
+        ? renderFrame(element, index, `c${element.id}`)
+        : renderComponent(element, index, `c${element.id}`)
```

Comment:
I have checked this one and unfortunately we can't setup id here because it overrides content element templates. 
So when developer setup `:id=""` in their content elements template, then it will be overrided by this line.

Reply:
At each content element it makes more sense, because that is always there and the frame not.

What do you think of this solution? [Fluid Styled Content Default Layout](https://github.com/TYPO3-CMS/fluid_styled_content/blob/main/Resources/Private/Layouts/Default.html)

Reply:
I see, they are adding empty anchor in case of no frame, but I'm not sure about this solution with empty anchor. From HTML perspective is valid, but from a11y it could be problematic. 
However I have checked it again and we can go with setup ID on `T3Dynamic` - developer will be able to override it, but they have use https://v2.vuejs.org/v2/api/#inheritAttrs